### ISSUE TITLE
Add support for responsiveness without reload

### ISSUE THREAD

**Issue by JulianJorgensen** [Prinzhorn/skrollr-stylesheets#29]
@Prinzhorn After 2 days of hard work, I've now come with a simple solution on how to support responsiveness using both skrollr and skrollr-stylesheets!

Please see my forks for further info:
https://github.com/JulianJorgensen/skrollr
https://github.com/JulianJorgensen/skrollr-stylesheets

Note: the modified branches are meant to be used together, but can also be used individually.

**Comment by Prinzhorn** [comment]
Sounds great. I don't have time to look at it now, but I will.

Can you create pull requests for those? This way I can comment on the code (each line etc.)

**Comment by bartrail** [comment]
this really sounds great. area there any news?

**Comment by Prinzhorn** [comment]
> this really sounds great. area there any news?

No. And this diff doesn't help at all https://github.com/JulianJorgensen/skrollr-stylesheets/commit/68a5b9910ffda1b6e0cb6ab1e1e5cec465452691

**Comment by JulianJorgensen** [comment]
I can't remember how I got it responsive now, but it's working and live here:
http://goldenchai.com

**Comment by JulianJorgensen** [comment]
@Prinzhorn @bartrail  ^^

**Comment by evolve2k** [comment]
@JulianJorgensen "After 2 days of hard work, I've now come with a simple solution on how to support responsiveness using both skrollr and skrollr-stylesheets!"

I've just started using scrollr and this sounds fantastic!
Any chance you could have another look at sending across a pull request?

If it's really too far forgotten, maybe we should just close this issue...
